bitnami/common/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: common
 sources:
   - https://github.com/bitnami/charts
 type: library
-version: 2.14.0
+version: 2.14.1
```

Comment:
We didn't updated in https://github.com/bitnami/charts/pull/21595 but for `bitnami/common` -> `appVersion=version`.